src/inmanta/const.py

```diff
@@ -286,3 +286,11 @@ class DesiredStateVersionStatus(str, Enum):
     candidate = "candidate"
     retired = "retired"
     skipped_candidate = "skipped_candidate"
+
+
+class NotificationSeverity(str, Enum):
+    message = "message"
+    info = "info"
```

Comment:
What is the difference between `message` and `info`? Intuitively they sound the same to me. In case they are not the same things, we should come up with a better name to clearly indicate the difference between both.